examples/Misc/UploadMediaBundle.php

```diff
@@ -0,0 +1,134 @@
+
+        // Generate a refreshable OAuth2 credential for authentication.
+        $oAuth2Credential = (new OAuth2TokenBuilder())->fromFile()->build();
+
+        // Construct a Google Ads client configured from a properties file and the
+        // OAuth2 credentials above.
+        $googleAdsClient = (new GoogleAdsClientBuilder())
+            ->fromFile()
+            ->withOAuth2Credential($oAuth2Credential)
+            ->build();
+
+        try {
+            self::runExample(
+                $googleAdsClient,
+                $options[ArgumentNames::CUSTOMER_ID] ?: self::CUSTOMER_ID
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     */
+    public static function runExample(GoogleAdsClient $googleAdsClient, int $customerId)
+    {
+        // Creates an HTML5 zip file media bundle content.
+        $bundleContent = file_get_contents(self::BUNDLE_URL);
+
+        // Creates a media file.
+        $mediaFile = new MediaFile([
+            'name' => new StringValue(['value' => 'Ad Media Bundle']),
+            'type' => MediaType::MEDIA_BUNDLE,
+            'source_url' => new StringValue(['value' => self::BUNDLE_URL]),
+            'media_bundle' => new MediaBundle(['data' => $bundleContent])
+        ]);
+
+        // Creates a media file operation.
+        $mediaFileOperation = new MediaFileOperation();
+        $mediaFileOperation->setCreate($mediaFile);
+
+        // Issues a mutate request to add the media file.
+        $mediaFileServiceClient = $googleAdsClient->getMediaFileServiceClient();
+        $response = $mediaFileServiceClient->mutateMediaFiles(
+            $customerId,
+            [$mediaFileOperation]
+        );
+
+        printf("Added %d media bundles:%s", $response->getResults()->count(), PHP_EOL);
+
+        foreach ($response->getResults() as $addedMediaFile) {
+            /** @var MediaFile $addedMediaFile */
+            printf(
+                "New media bundle added with resource name: '%s'%s",
```

Comment:
Nit: Let's use a sentence here "... bundle was added with ..."

Reply:
I switched to a single message log.